quiz.js

```diff
@@ -0,0 +1,46 @@
+// Create a tree function that accepts a single object as an argument.
+// The object has two key/value pairs from user input in a <input type="text"> field in the DOM.
+// Tree should be shown in the console.
+
+function tree() {
+  var height = document.getElementById("height").value;
+  var char = document.getElementById("char").value;
+
+  for (var i = 0; i < height; i++) {
+    var str = ''; var space = (height - i);
+    if (i == 0) {
+    }
+
+    str += ' '.repeat(space + 1);
+    var zero = 2 * i + 1;
+    str += char.repeat(zero);
+
+    console.log(str);
+  }
+}
+
+// User can either press the enter key or click a button that is labeled "Grow your tree".
+// Display an alert stating that both fields must have a value.
+
+function inputAlert() {
+  var rows = document.getElementById("height").value;
+  var leaves = document.getElementById("char").value;
```

Comment:
make this into an object:
var tree = {
rows:  document.getElementById("height").value,
leaves: document.getElementById("char").value
}